nondex-maven-plugin/src/it/clean-it/pom.xml

```diff
@@ -38,11 +35,6 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>@project.groupId@</groupId>
-      <artifactId>nondex-annotations</artifactId>
-      <version>@project.version@</version>
```

Comment:
Yes. They never needed it. They were added and were masking a bug that is exposed when they are not present. I added other ITs that have annotations and kept these ones simpler.